### DISCUSSION TITLE
The Keynote is outdated

### DISCUSSION THREAD

**Discussion by Astrrra** [djsime1/awesome-flipperzero#5] [General]
The keynote in the last section was made for the December 2021 hackathon, and is quite old now. I'd recommend at least adding a note about this, as right now it can easily confuse people.

**Comment by djsime1** [comment]
Thanks for letting me know. Is there a new keynote available? Because if so, I'll definitely replace it. Otherwise, I already had an update I was going to push tonight, so I'll add a note to the entry.
